### ISSUE TITLE
How to configure osTicket to fetch emails from a specific date?

### ISSUE THREAD

**Issue by alirezastack** [osTicket/osTicket#2722]
When I configured email address of department in order to fetch emails from my inbox, osTicket fetched emails for about 2 years ago and created tickets for all of them, isn't there a way to say fetch from a specific date or so?

**Comment by Hannibal226** [comment]
Why u don't go into inbox, create a folder below (e.g. "old mail") move all old into that.
After this starting the fetch?

As I understand u correct it also isn't nessecary that u keep all of them directly in inbox,
so why so complicated? :P 

CHEERS!

**Comment by ntozier** [comment]
osTicket fetches all email in the mail box.
The only ways to prevent that would be to 1. delete the old messages. or 2. move them to a sub folder.

**Comment by Finn10111** [comment]
@ntozier does your suggested solution really work? I am planning to use osTicket and I am working with a test mail box at the moment. I disabled to cron, wrote some mails, moved them to the sub folder and enabled the cron again. Unfortunately the mails in the subfolder caused osTicket to create tickets. Is this the expected behaviour? 

> The only ways to prevent that would be to 1. delete the old messages. or 2. move them to a sub folder.

I was wondering why this issue is still open so I decided to reply instead of creating a new issue because I don't see a clear answer or a working solution.
